include/aos/sm/networkmanager.hpp

```diff
@@ -399,6 +399,46 @@ class NamespaceManagerItf {
     virtual Error DeleteNetworkNamespace(const String& ns) = 0;
 };
 
+/**
+ * Link attributes.
+ */
+struct LinkAttrs;
```

Comment:
Where this struct will be defined? Or will be defined later?

Reply:
Yes, it will be defined in platform-specific code, because it will be different in each platform